File: 1768.py
def print_tronco(n):
  s = (n-1) // 2
  print(' '*s + '*')

  s = s-1
  print(' '*s + '***')

def print_folhas(n):
  s = (n-1) // 2
  f = 1


  while f < n:
    print(' '*s + '*'*f)
    s = s-1
    f = f+2

  print('*'*f)


while True:
  try:
    n = int(input())

    print_folhas(n)
    print_tronco(n)
    print()
  except EOFError:
    break

File: 1768.c
#include <stdio.h>

void print_folhas(int n) {
  int s, f;
  s = (n-1) / 2;
  f = 1;

  while (f < n) {
    for (int i = 0; i < s; i++) printf(" ");
    for (int i = 0; i < f; i++) printf("*");
    printf("\n");

    s = s-1;
    f = f+2;
  }

  for (int i = 0; i < f; i++) printf("*");
  printf("\n");
}

void print_tronco(int n) {
  int s;
  s = (n-1) / 2;

  for (int i = 0; i < s; i++) printf(" ");
  printf("*\n");

  s = s-1;

  for (int i = 0; i < s; i++) printf(" ");
  printf("***\n");
}

int main() {
  int n;

  while (scanf("%d", &n) != EOF) {
    print_folhas(n);
    print_tronco(n);
    printf("\n");
  }

  return 0;
}